### DISCUSSION TITLE
[Issue]:Request for PR Review

### DISCUSSION THREAD

**Discussion by dev-madhurendra** [dev-madhurendra/OOPs#37] [General]
Hi @Suryac72 ,
I'd like to request your input and feedback on the pending Pull Request in our [OOPs](https://github.com/dev-madhurendra/OOPs) repository. Your insights and suggestions are valuable to ensure the success of the pending PRs.

Thank You !

**Comment by Suryac72** [comment]
Hi @dev-madhurendra, Feedback for OOPs repository.

1) comments should be short and crisp. try to write self-explanatory code
2) Do not use any resource links. try to use tools to construct sentences in a professional way on their own.

These are inputs from my end.

**Reply by dev-madhurendra** [reply]
Thank You @Suryac72 
Keep Reviewing Keep Contributing.....
